app/controllers/hub/efile_submissions_controller.rb

```diff
@@ -17,9 +17,10 @@ def index
     # a little bit unexpectedly, the "show" page actually uses the client id to load the client. Then,
     # loops through the tax_returns that have efile_submissions.
     def show
-      @client = Client.find(params[:id])
-      authorize! :read, @client
-      @tax_returns = @client.tax_returns.joins(:efile_submissions).uniq # get all tax returns with submissions
+      client = Client.find(params[:id])
```

Comment:
This fixes the broken efile_submissions page that has not worked since we archived intakes.